recipe/meta.yaml

```diff
@@ -162,7 +161,7 @@ outputs:
         - async-timeout
         - colorful
         - gpustat >=1.0.0
-        - grpcio
+        - grpcio >=1.56,!=1.58.*
```

Comment:
Don't we have a libgrpc migration pin to 1.59 that will make this irrelevant?

Reply:
1.59 has been rolled out to essentially all of conda-forge, 1.60 is available but probably not installable broadly due to not having been migrated yet (though that's coming up soon)

Reply:
Right, I was referring to the libgrpc pin from the migration added in #140

Reply:
I agreee with Mattip. I believe that since you have tried to use a vendored version of 1.59 you should also pin to 1.59 to avoid future issues.

Reply:
@hmaarrfk since libgrpc is [pinned to 1.59](https://github.com/conda-forge/ray-packages-feedstock/blob/main/.ci_support/migrations/libgrpc159_libprotobuf4244.yaml) via the migration, and I would assume libgrpc is a dependency of grpcio, do we need pinning here at all?

Reply:
This pin is now irrelevant, grpcio pins libgrpc and ray-core now pins libgrpc. We can revert this line.

Reply:
I removed the pin.